core/server/common/src/main/java/alluxio/master/journal/AbstractJournalSystem.java

```diff
@@ -42,6 +56,60 @@ public synchronized void stop() throws InterruptedException, IOException {
     stopInternal();
   }
 
+  @Override
+  public void addJournalSink(Master master, JournalSink journalSink) {
+    mSinkLock.writeLock().lock();
+    try {
+      mJournalSinks.compute(master.getName(), (key, entry) -> {
```

Comment:
Since we're locking externally, this can be done in one line with
```java
mJournalSinks.computeIfAbsent(master.getName(), (k, v) -> new HashSet<>()).add(journalSink);
```

Reply:
Updated. 
I forgot to change this after I added the external locking.